gis-data-sharing/open-data/index.md

```diff
@@ -6,16 +6,53 @@ author:
+
+First, you have to be a member of the SGID groups that are the best fit for your datasets. Reach out to [AGRC staff]({% link about/contact/index.html %}) to request an invite. We'll work with you to identify what datasets are good matches for the SGID Open Data site and what group they best fit in.
+
+Once you're a member of the groups, share each approved item to the appropriate group. You also need to make sure the item is shared with "Everyone (public)". Otherwise, it will only be visible to you (or other members of your organization if shared to the organization) in Open Data.
+
+## Presentation Tips
+
+![An annotated Open Data item]({% link images/od_card_anno.png %}){: .flex .flex--center }
+
+- The **title** is the first place people look for information. Because items in Open Data are also ArcGIS Online items that are searchable by anyone in the world, we have added "Utah" to the beginning of every title of our public ArcGIS Online items to provide better context.
+- A **user name** that includes the department/division's name or function helps users know where the dataset is coming from. For example, all the datasets we share are owned by the `UtahAGRC` user.
+- Open Data displays the text from the `Description` field of the source ArcGIS Online item for the **description**. You shouldn't count on any information in the ArcGIS Online `Summary` field to show up unless it's repeated in the description.
+- To help with searching, we include the "AGRC" and "SGID" **tags** on all our items while removing any tags that are repeated in the item's title. Please refer to Esri's [post about using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more ideas and information.
+- The "Last Updated" **date** is based on the last time the _ArcGIS Online item_ was changed, not the _data itself_. Things like updating the item's description or changing the thumbnail may update this date. For some frequently-updated datasets, we include the last update date in the description text or refer to the layer's description page on [gis.utah.gov/data]({% link data/index.html %}).
+- Keeping your **data schema** constant is a real help to your users. Some will be using your feature services directly in their webmaps rather than self-hosting a copy of your data and trying to keep it up-to-date. Removing fields or changing their types can break popups, search widgets, and other functionality. Adding new fields is less dangerous.
+- By default, users can't download spatial datasets from ArcGIS Online/Open Data as **File Geodatabases**. To enable this, go to the item's page in ArcGIS Online and click on the "Settings" tab. Scroll to the bottom and check the box to "Allow others to export data to different formats".
+  - NOTE: This will also allow people to export/copy some or all of the features if they add the dataset as a layer in ArcGIS Pro.
+
+## Taming the Sources List
```

Comment:
**thought**: this whole section seems like it will be more of an implementation detail once the sources page is created and we have a better way to deal with finding sources. While I agree that folks should be careful with their sources, I wonder if this is too much information for a small reward.

Reply:
The in-depth description is driven by a conversation I had with Zach and trying to explain to another agency why they weren't showing up in the sources list they way they thought they should. It felt like one of those things that it's better to write once and refer people to rather than repeat multiple times in individual emails.

Reply:
That is good to know. I wouldn't think agencies would have noticed.